src/test/java/teammates/test/cases/webapi/UpdateFeedbackResponseActionTest.java

```diff
@@ -449,15 +497,58 @@ protected void testAccessControl_studentAccessOwnResponse_accessible() {
 
         loginAsStudent(student1InCourse1.getGoogleId());
 
-        assertEquals(student1InCourse1.getEmail(), typicalResponse.getGiver());
+        assertEquals(student1InCourse1.getEmail(), student1ResponseToStudent1.getGiver());
 
-        FeedbackResponseUpdateRequest updateRequest = getUpdateRequest(typicalResponse.getRecipient());
+        FeedbackResponseUpdateRequest updateRequest = getUpdateRequest(student1ResponseToStudent1.getRecipient());
 
         String[] studentAccessOwnPersonParams = new String[] {
                 Const.ParamsNames.INTENT, Intent.STUDENT_SUBMISSION.toString(),
-                Const.ParamsNames.FEEDBACK_RESPONSE_ID, typicalResponse.getId(),
+                Const.ParamsNames.FEEDBACK_RESPONSE_ID, student1ResponseToStudent1.getId(),
         };
 
         verifyCanAccess(updateRequest, studentAccessOwnPersonParams);
     }
+
+    @Test
+    protected void testAccessControl_instructorSubmitStudentResponse_accessible() {
+
+        ______TS("Student intends to access own response, but logged in as instructor, should be accessible");
```

Comment:
A bit of a renaming, since this effectively comes down to a permission issue
```suggestion
        ______TS("Instructor attempts to edit student's response with appropriate permission, should be accessible");
```